package.json

```
@@ -38,6 +38,7 @@
     "axios": "1.3.3",
     "classnames": "2.3.2",
     "formik": "2.4.3",
+    "get-youtube-id": "^1.0.1",
```

```suggestion
    "get-youtube-id": "1.0.1",
```

Зависимости фиксируем. Нужно будет ещё раз выполнить `npm i`, чтобы обновился package-lock.json